### PR TITLE
Adds yet another Traitor Loadout

### PR DESCRIPTION
**Pull request by dawsonkeyes**
## About The Pull Request

Behold the newest in technology: a kit harnessing the raw unmatched energy of the monochrome ocean itself. The Greytide loadout features equipment that benefits preparation and lategame battles: your only weapon is the weak but still potent spear of the tide itself, but you have enhanced staying power with energy weapons and the ability to craft more powerful pipeguns and defense emplacements such as trash cannons- though none of the materials are provided to you!

## How This Contributes To The Skyrat Roleplay Experience

This loadout is an entirely different playstyle for traitors, benefiting those that lie in wait or construct massive defenses rather than all the others, which focus on espionage and full-frontal assault. You have the tools to get what you need, and are far more geared towards a battle on your own turf than an average traitor might be at the cost of a lack of early game versatility.

TLDR: encourages different traitor playstyles

## Changelog

:cl:
add: New traitor loadout for 30tc
/:cl:

The total contents of this kit are:
Greytide Spear
Pipegun Prime Crafting Book
Trash Cannon Crafting Book
Advanced RCD
3 Large Matter Cartridges
Ancient Jumpsuit
Gas Mask
3 Filters
Chameleon Toolbelt
Syndicate Toolbox
Noslip Shoes
Authentic Insulated Gloves
Big Sunglasses
6 Greytide Posters

If the cost needs adjustment up or down from 30, tell me

### CODE REVIEW

**Comment by dawsonkeyes** [comment]
considering adding an RCD with no matter cartridges but idk about the cost

**Comment by YakumoChen** [comment]
The spear is worth 20 TC by itself (or more. It'd an admin only weapon for a pretty good reason) It's essentially a reactive stealth armor and a weapon on one, and the clones also attack, and I don't think there's a cooldown. It's pretty ridiculous.

The bundle is way overtuned. I could compare it to the dark lord bundle but that gets one less spell book and a lot less goodies.

I would say remove both spell books, they are way extraneous and don't actually sync with the recipe books in any way.

**Comment by YakumoChen** [comment]
Unrelated nitpick but you should replace the syndie toolbox with the plastit one, since it comes with yellow insuls anyway

**Comment by dawsonkeyes** [comment]
the spear does have a cooldown and normal spear damage, and unlike the stealth armor it's a percentage chance to spawn a comrade clone
also i will totally replace the toolbox thankies

**Comment by dawsonkeyes** [comment]
> Unrelated nitpick but you should replace the syndie toolbox with the plastit one, since it comes with yellow insuls anyway

oh, by the way, plastit i think is either specifically given it's contents in the uplink code itself, or i just cant find the stupid path
you were right about the spellbooks though, that's more for novelty kits (cyborg ninja, dark lord) when greytide is about building and attrition

File: modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm [hunk 1]
```diff
@@ -248,3 +248,22 @@
 	new /obj/item/grenade/spawnergrenade/manhacks(src)
 	for(var/i in 1 to 6)
 		new /obj/item/pen/edagger(src)
+
+/obj/item/storage/box/syndie_kit/loadout/greytide/PopulateContents() //hope you like reading baby, this kit's all about little initial power for massive endgame potential- you have staying power with energy weapons and other armaments but need to find or make them yourself, so GET TIDING
+	new /obj/item/spear/grey_tide(src)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm:253]: I feel like this could be just a plasmaglass spear rather than this fancy spear that's admin-only.

File: modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm [hunk 2]
```diff
@@ -248,3 +248,22 @@
...
+	new /obj/item/book/granter/crafting_recipe/pipegun_prime(src)
+	new /obj/item/book/granter/crafting_recipe/trash_cannon(src)
+	new /obj/item/construction/rcd/loaded/upgraded(src)
+	for(var/i in 1 to 3)
+		new /obj/item/rcd_ammo/large(src)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm]: No need for extra ammo, a good greytider knows where to find iron in large quantity.

File: modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm [hunk 3]
```diff
@@ -248,3 +248,22 @@
...
+	new /obj/item/clothing/under/color/grey/ancient(src)
+	new /obj/item/clothing/mask/gas(src)
+	for(var/i in 1 to 3)
+		new /obj/item/gas_filter(src)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm]: This is not really necessary to this kit and will just fill the box unnecessarily.

File: modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm [hunk 4]
```diff
@@ -248,3 +248,22 @@
...
+	new /obj/item/storage/belt/chameleon(src)
+	new /obj/item/storage/toolbox/mechanical/old/clean(src) //contains insuls for the aspiring god of tiding
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm:260]: This is the toolbox that costs 2 TC and deals more damage the more TC it has in it. This doesn't make much sense if there's not really much TC left for the traitor after the fact, and it feels conflicting with the spear.

File: modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm [hunk 5]
```diff
@@ -248,3 +248,22 @@
...
+	new /obj/item/clothing/shoes/chameleon/noslip(src)
+	new /obj/item/clothing/glasses/sunglasses/big(src)
+	for(var/i in 1 to 6)
+		new /obj/structure/sign/poster/contraband/grey_tide(src) //:)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm]: Make it 5, I'd say make it less but at the same time, I agree with what you're going for.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
A couple nitpicks here and there.

File: modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm (hunk 5 above)
Comment (dawsonkeyes) [reply modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm]: good idea

File: modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm (hunk 4 above)
Comment (dawsonkeyes) [reply modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm:260]: good idea

File: modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm (hunk 3 above)
Comment (dawsonkeyes) [reply modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm]: true

File: modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm (hunk 2 above)
Comment (dawsonkeyes) [reply modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm]: good idea

File: modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm (hunk 1 above)
Comment (dawsonkeyes) [reply modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm:253]: Well, the spear is the kit's sole novelty item, in total fairness. The vibroblade was in a similar admin-only state until also granted to the Cyborg Ninja loadout. It's not as powerful as one might imagine- I've run the tests and the like, and it mostly shines in group combat rather than being focused down by three seccies.

File: modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm (hunk 4 above)
Comment (dawsonkeyes) [reply modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm:260]: however, the whole point is that the toolbox is the only one that comes with a full toolset plus insulated gloves, i understand the tc gimmick but it need not apply here

File: modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm (hunk 1 above)
Comment (GoldenAlpharex) [reply modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm:253]: It's still a flat 50 percent chance on hit to create a duplicate, from what I've noticed, which is still kinda strong. If it's the main piece of the whole thing, then it should be the main focus too.

File: modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm (hunk 4 above)
Comment (GoldenAlpharex) [reply modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm:260]: You can make a new toolbox like that, that doesn't have the whole TC functionality.

Comment (dawsonkeyes) [reply modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm:260]: Why make a whole new subtype just for that? The cost doesn't leave room anyways

File: modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm (hunk 1 above)
Comment (dawsonkeyes) [reply modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm:253]: It is strong, but in testing it's unable to beat more than a lone security officer, and for the high price of 30 telecrystals I think it's fair. If I removed it, I'd lower it to 20 or less.

Comment (thestubborn) [reply modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm:253]: maybe make it a subtype of the plasma spear that looks like the admin one?

Comment (dawsonkeyes) [reply modular_skyrat/modules/moretraitoritems/code/game/objects/items/storage/syndicate_loadout.dm:253]: thing about the plasma spear is that it's not a subtype, it's a variable difference in icon and the only change is that it has slightly higher thrown force (23)

**Comment by dawsonkeyes** [comment]
Fuck it I need to work on this some more
